docs/logging-and-monitoring/monitoring-in-rust.md

```diff
@@ -5,15 +5,15 @@ The goal of recording tracing information is twofold:
 - aid the debugging by recording low-severity events and spans, like recording
   function calls, their parameters, and so on
 
-## Summary
```

Comment:
In case anyone else is intrigued by this change :D it was an UTF-8 non-breaking space (`C2 A0`).